docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md

```diff
@@ -122,7 +122,7 @@ In this step, you set up a type provider with a data connection and obtain a dat
 type private EntityConnection = SqlEntityConnection<ConnectionString="Server=SERVER\InstanceName;Initial Catalog=School;Integrated Security=SSPI;MultipleActiveResultSets=true",Pluralize = true>
 ```
 
-  This action sets up a type provider with the database connection that you created earlier. The property `MultipleActiveResultSets` is needed when you use the ADO.NET Entity Framework because this property allows multiple commands to execute asynchronously on the database in one connection, which can occur frequently in ADO.NET Entity Framework code. For more information, see [Multiple Active Result Sets (MARS)](http://go.microsoft.com/fwlink/?LinkId=236929).
+  This action sets up a type provider with the database connection that you created earlier. The property `MultipleActiveResultSets` is needed when you use the ADO.NET Entity Framework because this property allows multiple commands to execute asynchronously on the database in one connection, which can occur frequently in ADO.NET Entity Framework code. For more information, see [Multiple Active Result Sets (MARS)](http://msdn.microsoft.com/library/ms131686.aspx).
```

Comment:
This URL actually redirects to https://docs.microsoft.com/en-us/sql/relational-databases/native-client/features/using-multiple-active-result-sets-mars. 
So the right new link would be /sql/relational-databases/native-client/features/using-multiple-active-result-sets-mars

Reply:
Would a double check on the redirect be a good idea to fix situations like this?

Reply:
I'm double checking them. No worries.

Reply:
Ive found a way to get these final links from the FWLinks, updating to accommodate with a repeated check on final redirect location.